blazor/datagrid/filter-menu.md

```diff
@@ -327,4 +327,101 @@ In the following sample, the `SfDateRangePicker` component is rendered in the fi
 }
 ```
 
-> [View Sample in GitHub.](https://github.com/SyncfusionExamples/blazor-datagrid-filtering-using-date-range-picker)
\ No newline at end of file
+> [View Sample in GitHub.](https://github.com/SyncfusionExamples/blazor-datagrid-filtering-using-date-range-picker)
+
+## Filter by multiple keywords using filter menu
+
+By default, the filtering action is performed based on the single value selected from the built-in component of the filter menu dialog. Now the DataGrid has an option to perform filtering actions based on multiple values. This can be achieved by using the [FilterTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn_FilterTemplate) feature and [FilterByColumnAsync](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.SfGrid-1.html#Syncfusion_Blazor_Grids_SfGrid_1_FilterByColumnAsync_System_String_System_String_System_Object_System_String_System_Nullable_System_Boolean__System_Nullable_System_Boolean__System_Object_System_Object_) method of the Grid.
+
+In the following sample, the MultiSelect component is rendered in the filter menu of the Customer Name column. Using the ValueChange event of `SfMultiSelect`, the selected values from the multiselect dropdown can be retrieved as a list. In the [OnActionBegin](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridEvents-1.html#Syncfusion_Blazor_Grids_GridEvents_1_OnActionBegin) event of the Grid, when `RequestType` is `Filtering`, the list(selected values) can be passed to the `FilterByColumnAsync` method to get the filtered records.
```

Comment:
In the following sample, the MultiSelect component is rendered in the filter menu of the Customer Name column. Using the ValueChange event of `SfMultiSelect,` the selected values from the multiselect dropdown can be retrieved as a list. In the [OnActionBegin](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridEvents-1.html#Syncfusion_Blazor_Grids_GridEvents_1_OnActionBegin) event of the Grid, when `RequestType` is `Filtering,` the list(selected values) can be passed to the `FilterByColumnAsync` method to get the filtered records.